ami/flowchart/library/DisplayWidgets.py

```diff
@@ -567,7 +568,9 @@ def __init__(self, topics=None, terms=None, addr=None, parent=None, **kwargs):
         self.flip = False
         self.rotate = 0
         self.log_scale_histogram = False
-        self.auto_levels = True
+        self.auto_levels = False
+        self.ratio = 1 # need to see how to get the right aspect ratio from data
```

Comment:
Can we get that number from the data somehow?

Reply:
As discussed on slack, let's just hardcode the ratio to 1 for now. We might never need another value.